tools/genesis-snapshot/presets/presets.go

```diff
@@ -21,7 +21,7 @@ var Base = []options.Option[snapshotcreator.Options]{
 	snapshotcreator.WithProtocolParameters(
 		iotago.NewV3ProtocolParameters(
 			iotago.WithNetworkOptions("default", "rms"),
-			iotago.WithSupplyOptions(1_000_0000, 100, 1, 10),
+			iotago.WithSupplyOptions(10_000_000_000_000, 100, 1, 10),
```

Comment:
Isn't this overridden by the WIthSupplyOptions of the more specific Docker preset?